src/_data/side-nav.yml

```diff
@@ -144,8 +144,8 @@
           permalink: /web/js-interop
     - title: Google APIs
       permalink: /guides/google-apis
-    - title: Mobile apps
-      permalink: /mobile
+    - title: Apps
+      permalink: /apps
```

Comment:
I'm not crazy about this title — especially when compared to the next couple of titles. It seems weird to have just "Apps" followed by two "Xyz apps" entries. Maybe we could do "Flutter apps"? Or "Multi-platform apps"?

![image](https://user-images.githubusercontent.com/2164483/128391922-2983529b-b5cc-42a6-a4a9-d712a811a8fa.png)

Reply:
I think Multi-platform apps sounds great and will help others find this page if they perhaps don't know what Flutter means.

Reply:
Changed to 'Multi-platform apps'